tests/integration/mask.bats

```diff
@@ -9,16 +9,16 @@ function setup() {
 	# Create fake rootfs.
 	mkdir rootfs/testdir
 	echo "Forbidden information!" > rootfs/testfile
-
-	# add extra masked paths
-	update_config '(.. | select(.maskedPaths? != null)) .maskedPaths += ["/testdir", "/testfile"]' 
```

Comment:
Why?

Reply:
I guess because I added the umask test in this file, so updating masked paths only should happen in the mask path file?

Reply:
Umask is unrelated to maskpaths, please add the umask test in a separate file.